frontend/packages/shared/src/components/InputActionWrapper/InputActionWrapper.test.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { fireEvent, render as rtlRender, screen } from '@testing-library/react';
+import type { InputActionWrapperProps } from './InputActionWrapper';
+import { InputActionWrapper } from './InputActionWrapper';
+
+jest.mock('./InputActionWrapper', () => ({
+  __esModule: true,
+  InputActionWrapper: ({ children, onEditClick, onDeleteClick, onSaveClick }: any) => (
+    <div>
+      {children}
+      <button onClick={onEditClick} aria-label='general.edit'>
+        Edit
+      </button>
+      <button onClick={onSaveClick} aria-label='general.save'>
+        Save
+      </button>
+      <button onClick={onDeleteClick} aria-label='general.delete'>
+        Delete
+      </button>
+    </div>
+  ),
+}));
```

Comment:
Should we  need to use jest.mock in this scenario? 🤔 I believe we can effectively test the component by rendering the <InputActionWrapper> within the render method, following the same approach we use for testing other components. What do you think? 😊

Reply:
Agree 😊,  I removed and updated the test